### PR TITLE
Docs cleanups: Part 2

### PR DESCRIPTION
**Pull request by sjspielman**
More cleanups! I think this should be pretty straightforward, but let me know if any changes I made are mysterious.

Note that I did end up having to replace an image as well to get around the STUB_LINK still in there (and also wonky grammar in the PR title). I made a note about this here too: https://github.com/AlexsLemonade/OpenScPCA-analysis/issues/278#issuecomment-2032486853

### CODE REVIEW

**Comment by sjspielman** [comment]
@jaclyn-taroni I want to draw your attention to this commit - https://github.com/AlexsLemonade/OpenScPCA-analysis/pull/335/commits/a7e976fd861a5060ddf524a156e5d2ac3ee3a8b6

When I was testing the links here, I found the TOC on this page does not work in the same way as the corresponding CoC in GitHub works, because both the page is called code of conduct, and a section is called that too. Since the website displays a TOC on the right-hand side anyways, I removed the TOC from the website from that version of the document. Was this ok or should I restore it?

File: docs/communications-tools/github-issues/what-makes-a-good-issue.md [hunk 1]
```diff
@@ -8,9 +8,9 @@ Good issues include the following:
 - A title that is informative about the contents of the issue at a glance
 - A relatively small and focused scope, such that [the pull request that closes them is appropriate in scope](../../contributing-to-analyses/creating-pull-requests/scoping-pull-requests.md)
 - Ideas about how to accomplish a segment of work
-- Enough context...
+- Enough context so that:
     - For someone other than the issue author to accomplish the work
```
Comment (allyhawkins) [inline docs/communications-tools/github-issues/what-makes-a-good-issue.md:12]:
```suggestion
    - Someone other than the issue author to accomplish the work
```
I think this needs to be changed to match the wording you changed before this.

File: docs/contributing-to-analyses/working-with-git/staying-in-sync-with-upstream.md [hunk 2]
```diff
@@ -23,28 +23,31 @@ Follow these steps to keep your fork in sync with changes in `AlexsLemonade/Open
...
 
     You can then press `Sync fork` to sync your fork with any upstream changes.
 
     <figure markdown="span">
         ![Sync fork](../../img/upstream-sync-1.png){width="600"}
     </figure>
 
-1. After pressing `Sync fork` a pop-up will notify you that the branch is out-of-date.
-    Go ahead and press `Update branch`.
+2. After pressing `Sync fork`, GitHub will notify you that the branch is out-of-date.
```
Comment (allyhawkins) [inline docs/contributing-to-analyses/working-with-git/staying-in-sync-with-upstream.md:34]:
```suggestion
1. After pressing `Sync fork`, GitHub will notify you that the branch is out-of-date.
```

File: docs/contributing-to-analyses/working-with-git/staying-in-sync-with-upstream.md [hunk 3]
```diff
@@ -23,28 +23,31 @@ Follow these steps to keep your fork in sync with changes in `AlexsLemonade/Open
...
+
+    - If GitHub instead tells you there are conflict that must be resolved, see the [section below on resolving sync conflicts](#syncing-your-fork-when-there-are-sync-conflicts).
+
 
     <figure markdown="span">
         ![Update branch](../../img/upstream-sync-2.png){width="300"}
     </figure>
 
-1. If your `main` branch has been updated successfully, you will now see a note that the branch is up to date with `AlexsLemonade/OpenScPCA-analysis`.
+3. If your `main` branch has been updated successfully, you will now see a note that the branch is up to date with `AlexsLemonade/OpenScPCA-analysis`.
```
Comment (allyhawkins) [inline docs/contributing-to-analyses/working-with-git/staying-in-sync-with-upstream.md:44]:
```suggestion
1. If your `main` branch has been updated successfully, you will now see a note that the branch is up to date with `AlexsLemonade/OpenScPCA-analysis`.
```

File: docs/contributing-to-analyses/working-with-git/staying-in-sync-with-upstream.md [hunk 4]
```diff
@@ -23,28 +23,31 @@ Follow these steps to keep your fork in sync with changes in `AlexsLemonade/Open
...
 
     <figure markdown="span">
         ![Successful sync](../../img/upstream-sync-3.png){width="600"}
     </figure>
 
-1. Before making any changes locally, you will need to pull the synced changes from GitHub to your computer.
+4. Before making any changes locally, you will need to pull the synced changes from GitHub to your computer.
```
Comment (allyhawkins) [inline docs/contributing-to-analyses/working-with-git/staying-in-sync-with-upstream.md:50]:
```suggestion
1. Before making any changes locally, you will need to pull the synced changes from GitHub to your computer.
```

File: docs/policies/code-of-conduct.md [hunk 5]
```diff
@@ -6,24 +6,6 @@ Alex’s Lemonade Stand Foundation’s (ALSF) Childhood Cancer Data Lab (Data La
 We are committed to creating a friendly and respectful place for contributing and learning.
 Everyone participating in the Open Single-cell Pediatric Cancer Atlas (OpenScPCA) project is expected to show respect and courtesy to others.
 
-**Table of contents:**
```
Comment (allyhawkins) [inline docs/policies/code-of-conduct.md:9]: Did you mean to delete this?

**Review by allyhawkins — COMMENTED** [review]
Looks good, just a few small comments.

File: docs/policies/code-of-conduct.md (hunk 5 above)
Comment (sjspielman) [reply docs/policies/code-of-conduct.md:9]: I did, but - https://github.com/AlexsLemonade/OpenScPCA-analysis/pull/335#issuecomment-2032619737

**Comment by jaclyn-taroni** [comment]
> @jaclyn-taroni I want to draw your attention to this commit - https://github.com/AlexsLemonade/OpenScPCA-analysis/pull/335/commits/a7e976fd861a5060ddf524a156e5d2ac3ee3a8b6
> 
> 
> 
> When I was testing the links here, I found the TOC on this page does not work in the same way as the corresponding CoC in GitHub works, because both the page is called code of conduct, and a section is called that too. Since the website displays a TOC on the right-hand side anyways, I removed the TOC from the website from that version of the document. Was this ok or should I restore it?

Yes, that's fine.

File: docs/communications-tools/github-issues/what-makes-a-good-issue.md [hunk 6]
```diff
@@ -8,9 +8,9 @@ Good issues include the following:
 - A title that is informative about the contents of the issue at a glance
 - A relatively small and focused scope, such that [the pull request that closes them is appropriate in scope](../../contributing-to-analyses/creating-pull-requests/scoping-pull-requests.md)
 - Ideas about how to accomplish a segment of work
-- Enough context...
-    - For someone other than the issue author to accomplish the work
-    - To set up [pull request reviewers for success](../../contributing-to-analyses/pr-review-and-merge/index.md)
+- Enough context so that:
+    - Someone other than the issue author to accomplish the work
```
Comment (allyhawkins) [inline docs/communications-tools/github-issues/what-makes-a-good-issue.md:12]:
```suggestion
    - Someone other than the issue author can accomplish the work
```

File: docs/contributing-to-analyses/determining-requirements/determining-compute-requirements.md [hunk 7]
```diff
@@ -41,7 +41,7 @@ Although we are most concerned with finding the right size computer to run an an
 
 We generally will not ask you to report input file sizes because we can calculate them if we know what project(s) or sample(s) you are analyzing!
 
-However, if your module code generates large intermediate files, document that information in the README of your module to help others who want to run the code.
+However, if your module code generates large intermediate files, [document that information your module's `README.md` file](../analysis-modules/documenting-analysis.md) to help others who want to run the code.
```
Comment (allyhawkins) [inline docs/contributing-to-analyses/determining-requirements/determining-compute-requirements.md:44]:
```suggestion
However, if your module code generates large intermediate files, [document that information in your module's `README.md` file](../analysis-modules/documenting-analysis.md) to help others who want to run the code.
```

**Review by allyhawkins — APPROVED** [review]
LGTM

**Pull request merged**